src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  margin: 0;
+}
+
+div {
```

Comment:
do not add styles to div, it is too general, use classes for styling(add general class to each block)